### ISSUE TITLE
Why is deterministic signing required for ML-DSA?

### ISSUE THREAD

**Issue by sfluhrer** [tireddy2/ikev2-pqc-auth#10]
You state:

If ML-DSA is used as an authentication method within the IKEv2 protocol, the deterministic version of ML-DSA MUST be used.

Might I ask why this MUST statement is there (and not about SLH-DSA, which gives similar flexibility?

**Comment by sfluhrer** [comment]
Actually, between 'hedged' and 'deterministic' versions of ML-DSA/SLH-DSA are interoperable, and the hedged versions does not have a security issue, why does this draft make any mandate as to which is used?

**Comment by tireddy2** [comment]
> Actually, between 'hedged' and 'deterministic' versions of ML-DSA/SLH-DSA are interoperable, and the hedged versions does not have a security issue, why does this draft make any mandate as to which is used?

Agreed, modified text to say "hedged" mode is preferred. I don't re-collect why we chose "deterministic" mode, it was present from 00 version.
